traitsui/testing/tester/ui_wrapper.py

```diff
@@ -73,6 +73,44 @@ def __init__(self, target, registries, delay=0):
         self._registries = registries
         self.delay = delay
 
+    def help(self):
+        """ Print help messages.
+        (This function is intended for interactive use.)
+        """
+        interactions = dict()
+        locations = dict()
+        for registry in self._registries[::-1]:
+            for type_ in registry.get_interactions(self.target.__class__):
+                interactions[type_] = registry.get_interaction_doc(
+                    self.target.__class__, type_)
+
+            for type_ in registry.get_locations(self.target.__class__):
+                locations[type_] = registry.get_location_doc(
+                    self.target.__class__, type_)
+
+        print("Interactions")
+        print("------------")
+        pairs = sorted(interactions.items(), key=lambda value: repr(value[0]))
+        for interaction_class, doc in pairs:
+            print(repr(interaction_class))
+            print(
+                *("    " + line for line in doc.split("\n")),
+                sep="\n",
+            )
+            print()
+
+        print()
```

Comment:
instead of having empty print function calls which are adding a new line if i understand correctly, why not print
```
----------
Locations
----------
```

that way, you are emphasising the text and we won't need empty print calls. Same comment regd "Interactions".

Reply:
It was motivated by the situation when there are no interactions supported. If I did not have a newline, it would look like this, which is a little uglier.

```
Interactions
------------
Locations
---------
<class 'traitsui.testing.tester.locator.Index'>
    A locator for locating a target that is uniquely specified by a single
    0-based index.
    
    Attributes
    ----------
    index : int
        0-based index

Help on an item in the list editor
Interactions
------------
<class 'traitsui.testing.tester.command.MouseClick'>
    An object representing the user clicking a mouse button.
    Currently the left mouse button is assumed.
    
    In most circumstances, a widget can still be clicked on even if it is
    disabled. Therefore unlike key events, if the widget is disabled,
    implementations should not raise an exception.

```

I could change this so that there is at least a sentence for saying there is nothing. It adds a bit more logic but it might look nicer.

Reply:
Done. Now the example where there are no interactions look like this:
```
Interactions
------------
No interactions are supported.

Locations
---------
<class 'traitsui.testing.tester.locator.Index'>
    A locator for locating a target that is uniquely specified by a single
    0-based index.

    Attributes
    ----------
    index : int
        0-based index
```

Is the "-------" above the section still needed?

Reply:
nope. looks good.